### ISSUE TITLE
Run Status in Rules Report, engine v0.15

### ISSUE THREAD

**Issue by mhungria** [cdisc-org/cdisc-rules-engine#1639]
### Standard

CDISC SDTM/SDTMIG

### Reference Rule ID(s)

CG0225, CG0034, ...

### Conformance Rule ID(s) (if published in CORE)

CORE-000140, CORE-000168, ...

### CLI Command Used (if applicable)

python3 core.py validate -s SDTMIG -v 3.4 -dv 2.1 -dxp .../tempv/data/define.xml -vx y -dp .../tempv/data/unit-test-coreid-CG0555-negative.xlsx --filetype json -o .../tempv/data/val_reports/bash_coreval_CG0555_negative_01_data_mhungria_SDTMIGv3.4_2026-02-25 --output-format JSON --output-format XLSX -l info -p verbose_output

### Rule YAML

```YAML
```

### Attach any sample test data file(s)

[unit-test-coreid-CG0555-negative.xlsx](https://github.com/user-attachments/files/25557760/unit-test-coreid-CG0555-negative.xlsx)

[define.xml](https://github.com/user-attachments/files/25558048/define.xml)

### Attach any output report and/or log file(s)

[bash_coreval_CG0555_negative_01_data_mhungria_SDTMIGv3.4_2026-02-25.json](https://github.com/user-attachments/files/25557756/bash_coreval_CG0555_negative_01_data_mhungria_SDTMIGv3.4_2026-02-25.json)

<img width="636" height="144" alt="Image" src="https://github.com/user-attachments/assets/09882b18-41af-4d99-a5ae-31d42fc48bcf" />

<img width="533" height="146" alt="Image" src="https://github.com/user-attachments/assets/8e91f10f-dbf3-4f5e-8105-ac0df3c506e8" />

### Expected output

**For rules involving cross-datasets checks, when one of the datasets is not present, I would expect the run status for a rule be SKIPPED instead of EXECUTION ERROR.**
Please note I just provided a couple of rule references even when in the enclosed execution there are many more. The only dataset included in the data is pp.

**Release v0.15 documentation**:
The possible rule run statuses are:
    SUCCESS - The rule ran, data was validated, and no issues were reported.
    SKIPPED - The rule was unable to be run for one of the following reasons:
        Column not found in data
        Domain not found
        Schema validation is off
        Outside scope
    ISSUE REPORTED - The rule ran, data was validated, and issues were reported
    EXECUTION ERROR - The validation failed for an unknown reason caused by rule evaluation or execution. Error details are found in the Issue Details tab.

### Any Additional Information

Thanks for the new release v0.15!

Note: executing on a linux box.
